mbs_results/validation_checks.py

```diff
@@ -53,6 +59,40 @@ def colnames_clash(
     )
 
 
+def period_and_reference_not_given(
+    reference, period, responses_keep_cols, contributors_keep_cols, **config
+):
+    """
+    _summary_
```

Comment:
Summary missing

Reply:
Summary has now been provided for this function, nice spot